addons/point_of_sale/static/src/app/screens/scale_screen/scale_screen.js

```diff
@@ -75,4 +88,13 @@ export class ScaleScreen extends Component {
     get productUom() {
         return this.props.product?.uom_id?.name;
     }
+    async handleTareButtonClick() {
```

Comment:
don't forget to override this method in the scale_screen in pos_iot module. In that we replace the use of `hardwareProxy.readScale`.